mockkube/src/main/java/io/strimzi/test/mockkube/StatefulSetMockBuilder.java

```diff
@@ -126,7 +128,7 @@ private void mockScale(String resourceName, RollableScalableResource<StatefulSet
 
     @Override
     protected void mockCreate(String resourceName, RollableScalableResource<StatefulSet, DoneableStatefulSet> resource) {
-        when(resource.create(any())).thenAnswer(cinvocation -> {
+        when(resource.create(any(StatefulSet.class))).thenAnswer(cinvocation -> {
```

Comment:
So I'm guessing this is necessary because there's some new overloaded `create()` method which makes the old `any()` ambiguous. Can you tell me what the new `create()` method is?

Reply:
I believe it is caused by adding this: https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Createable.java#L22

Reply:
Ah, excellent, that's a method I requested myself actually. In the `AbstractResourceOperator` (and similar) we used to get "generic array creation" warnings because there was only a varargs method in `Creatable`. This means that we won't get those warnings now, so the `@SuppressWarnings` won't be needed any more. Can you hunt down those annotations and remove them? Perhaps in a followup PR?